src/lib/include/libsed.h

```diff
@@ -106,13 +106,28 @@ struct sed_tper_properties {
 	} property[NUM_TPER_PROPS];
 } __attribute__((__packed__));
 
+struct sed_blocksid_supported_feat {
+	uint8_t sid_valuestate:1;
+	uint8_t sid_blockstate:1;
+	uint8_t reserved1:6;
+	uint8_t hardware_reset:1;
+	uint8_t reserved2:7;
+	uint8_t reserved3[10];
+} __attribute__((__packed__));
+
 struct sed_opal_level0_discovery {
+	struct {
+		uint32_t feat_blocksid:1;
+		uint32_t reserved:31;
```

Comment:
Just to abide by the spec, we have MAX_FEATURES defined as 64. So lets use uint64_t here instead of uint32_t.